apps/hash-api/src/graph/index.ts

```diff
@@ -47,6 +48,52 @@ export const zeroedGraphResolveDepths: GraphResolveDepths = {
   hasRightEntity: { incoming: 0, outgoing: 0 },
 };
 
+export const currentTemporalAxes: QueryTemporalAxesUnresolved = {
```

Comment:
Maybe we want to add some doc-comments on these to quickly explain them in English, we should remember that this stuff isn't super intuitive and others (or us in the future) might come back and really struggle to quickly grasp what these do.

Maybe we say something like "Slices the datastore across this instant of time", and "According to the database's most up to date knowledge (transaction time), return the full history of entities and the times at which those decisions were made"

Reply:
I slightly refined the comments and added them, thanks for the suggestions! [Add documentation to constants](https://github.com/hashintel/hash/pull/2542/commits/892db700c238e01b3b4741e1420c4be6ec3d761d)